pkg/apis/core/annotation_key_constants.go

```diff
@@ -82,4 +82,10 @@ const (
 	//
 	// Not all cloud providers support this annotation, though AWS & GCE do.
 	AnnotationLoadBalancerSourceRangesKey = "service.beta.kubernetes.io/load-balancer-source-ranges"
+
+	// EndpointsLastTriggerTime is the annotation key, set for endpoints objects, that represents the
```

Comment:
I believe the answer is yes. This time will have a different semantics than the last-update-time (modification timestamp).
In the example below this would be set to T0, while the "modification time" would be T2.

Reply:
Yes - it's still needed. Because it is a way to somehow pass those information from pods to things that don't watch pods.